shell/platform/tizen/channels/app_control.cc

```diff
@@ -5,17 +5,69 @@
 #include "app_control.h"
 
 #include "flutter/shell/platform/tizen/channels/app_control_channel.h"
+#include "flutter/shell/platform/tizen/logger.h"
+
+intptr_t NativeInitializeDartApi(void* data) {
+  return Dart_InitializeApiDL(data);
+}
+
+int32_t NativeCreateAppControl(Dart_Handle handle) {
+  auto app_control = std::make_unique<flutter::AppControl>();
+  if (!app_control->handle()) {
+    return -1;
+  }
+  auto id = app_control->id();
+  Dart_NewFinalizableHandle_DL(
+      handle, app_control.get(), 64,
```

Comment:
Why is it 64? (Just curious.. :smile: )

Reply:
The value is a hint for the garbage collector as far as the documentation says. Increasing the size may increase the chance of finalizable objects to be collected.

As a side note, the general finalizer support will be added to the Dart language in the near future so this (finalization through FFI) is a temporary solution until then.